master/buildbot/process/properties.py

```diff
@@ -478,24 +478,7 @@ def getRenderingFor(self, properties):
 class Secret(_SecretRenderer):
```

Comment:
So is _SecretRenderer really necessary, then?

Reply:
I supposed it was used to differentiate between real `util.Secret` that comes from users and something that is created internally. `_SecretRenderer` is used in `_SecretIndexer`.